next.config.js

```diff
@@ -1,4 +1,9 @@
 /** @type {import('next').NextConfig} */
 module.exports = {
-  reactStrictMode: true,
+
+	reactStrictMode: true,
+	images: { domains: ['images.unsplash.com', 'cdn-icons-png.flaticon.com'] },
```

Comment:
instead of linking unsplash directly, can you download the image and put it in your fork?

probably the same thing with the icon